core/modules/server/routes/get-index.js

```diff
@@ -17,9 +17,10 @@ exports.method = "GET";
 exports.path = /^\/$/;
 
 exports.handler = function(request,response,state) {
-	var text = state.wiki.renderTiddler(state.server.get("root-render-type"),state.server.get("root-tiddler")),
-		responseHeaders = {
-		"Content-Type": state.server.get("root-serve-type")
```

Comment:
Add "Server-Timing" here, separate to two `var`

Reply:
It looks like there are changes on line 20 to drop the semicolon and line 21 to add the "var". Those sort of changes make the PR harder to follow and should be avoided unless there is a good reason.